src/components/Calculator.js

```diff
@@ -1,47 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
 import PropTypes from 'prop-types';
 import './Calculator.css';
+import calculate from '../logic/calculate';
 
-function Button({ value, className }) {
-  return <button type="button" className={className}>{value}</button>;
+function Button({ value, className, onClick }) {
+  return (
+    <button type="button" className={className} onClick={onClick}>
+      {value}
+    </button>
+  );
 }
 
 Button.propTypes = {
   value: PropTypes.string.isRequired,
   className: PropTypes.string,
+  onClick: PropTypes.func.isRequired,
 };
 
-// Default props for Button component
 Button.defaultProps = {
   className: '',
 };
```

Comment:
- It'd be awesome if you separated the `Button` component from the `Calculator` component. This leads to a cleaner code and a better organization. 🙏